tests/include/mir/test/doubles/mock_main_loop.h

```diff
@@ -55,6 +55,7 @@ class MockMainLoop : public MainLoop
     MOCK_METHOD1(unregister_fd_handler, void(void const*));
 
     MOCK_METHOD2(enqueue, void(void const*, ServerAction const&));
+    MOCK_METHOD(void, enqueue_with_guaranteed_execution, (ServerAction const&));
```

Comment:
Can you make this
`MOCK_METHOD(void, enqueue_with_guaranteed_execution, (ServerAction const&), (override));`?